tests/integration/vlan_test.py

```diff
@@ -64,6 +65,21 @@ def test_add_and_remove_vlan(eth1_up):
     assert not current_state[INTERFACES]
 
 
+def test_mac_addr_behavior_with_vlan(eth1_up):
```

Comment:
This name is not descriptive, it should specify the expected behavior (compare #368)